react/InputPassword/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import cx from 'classnames'
+import { default as LabelStyles } from '../Label/styles.styl'
+import Input from '../Input'
+import styles from './styles.styl'
+
+class InputPassword extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = { visible: false }
+    this.askShowHide = props.askShowHide !== false
+  }
+  toggleVisibility() {
+    this.setState({ visible: !this.state.visible })
+  }
+  render() {
+    return (
+      <div className={styles['wrapInputPassword']}>
+        {this.askShowHide && (
+          <div
+            className={cx(LabelStyles['c-label'], styles['actionOnInput'])}
+            onClick={() => this.toggleVisibility()}
+          >
+            {this.state.visible ? 'HIDE' : 'SHOW'}
```

Comment:
We cannot let untranslated text anywhere in our code.

Also, this line ask a question : we need to translate component from Cozy-UI and it has never been done before. A quick solution could be to pass the label as props (maybe `buttonHideLabel` and `buttonShowLabel`), waiting for a more global solution provided by Cozy-UI.

Reply:
Finally the `translate` signature could serve to something :) Default translating strings could be passed directly as options

```
export default translate({
  en: { show: 'Show', hide: 'Hide' },
  fr: { show: 'Montrer', hide: 'Cacher' }, 
})(InputPassword)
```